DataAcces.cs

```diff
@@ -1,17 +1,38 @@
-﻿using hotel.Models;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using hotel.Models;
+using System.Configuration;
+using System.Data;
+using System.Data.Sql;
+using System.Data.SqlClient;
 
 namespace hotel
 {
     public class DataAcces : IDataAcces
     {
+
+        private string ConnectionString = ConfigurationManager.ConnectionStrings["hotel.Properties.Settings.Setting"].ConnectionString;
         public void SaveCustomer(CustomerDto customerDto)
         {
-            throw new NotImplementedException();
+            using (var ConnectionSave = new SqlConnection(ConnectionString))
+            {
+                ConnectionSave.Open();
+
+                string insertQuery = "Insert into Customer(NameSurname, Telephone, Mail, Country, Tc, Address) values( @NameSurname, @Telephone, @Mail, @Country, @Tc, @Address)";
+                SqlCommand insertCommand = new SqlCommand(insertQuery, ConnectionSave);
```

Comment:
Try Catch ile kayıt işleminde hata olursa, hata mesajı vererek kaydı başa döndürmek gibi bir işlem eklesek mi sizce ?

Reply:
Iyi olabilir evet. SaveCustomer void yerine bir sonuc dondururse ona gore kontrollerini yapabilirsin. Ama acelesi yok onun sonradan da eklenebilir.

Reply:
sonrası için not olarak ekliyorum o halde.